### PR TITLE
sig-release: Add doc on version markers (migrated from k/test-infra)

### PR DESCRIPTION
**Pull request by justaugustus**
Part of https://github.com/kubernetes/sig-release/issues/850.

/assign @saschagrunert @cpanato @hasheddan @xmudrii 
cc: @kubernetes/release-engineering

### CODE REVIEW

**Review by hasheddan — APPROVED** [review]
/lgtm

File: contributors/devel/sig-release/kubernetes-versions.md [hunk 1]
```diff
@@ -0,0 +1,452 @@
...
+  - [Manually created jobs using generic version markers can be inaccurate](#manually-created-jobs-using-generic-version-markers-can-be-inaccurate)
+- [Previous Issues](#previous-issues)
+  - [linux/amd64 version markers are colliding with cross builds](#linuxamd64-version-markers-are-colliding-with-cross-builds)
+  - [Cross builds are stored in a separate GCS bucket](#cross-builds-are-stored-in-a-separate-gcs-bucket)
+  - [Generated jobs may not represent intention](#generated-jobs-may-not-represent-intention)
+  - [bazel version markers appear to be unused](#bazel-version-markers-appear-to-be-unused)
+
+## tl;dr
+
+You need a...
```
Comment (cpanato) [inline contributors/devel/sig-release/kubernetes-versions.md:49]: this is ok?

**Review by cpanato — APPROVED** [review]
/lgtm
just a comment, but that is nonblocking, I think I did not get the joke :)

/hold for a few more reviews if needed

**Comment by justaugustus** [comment]
Companion PR: https://github.com/kubernetes/test-infra/pull/19686

**Pull request merged**